motioneye/static/js/main.js

```diff
@@ -5215,6 +5229,10 @@ function doExitSingleViewCamera() {
     var cameraFrame = getCameraFrame(singleViewCameraId);
     var pageContainer = getPageContainer();
 
+    if (pageContainer.hasClass('single-cam-edit')) {
+        pageContainer.removeClass('single-cam-edit');
+    }
```

Comment:
I'm not sure whether checking for a class before removing it is beneficial performance wise (if the class is not currently assigned). I just see classes being removed without a check, usually. Of course it also depends on whether it is expected to be missing or present in most cases.

Reply:
You are right. Removed that.